huey/contrib/djhuey/management/commands/run_huey.py

```diff
@@ -67,8 +65,6 @@ def add_arguments(self, parser):
                     short = '-V'
                 if 'type' in kwargs:
                     kwargs['type'] = self._type_map[kwargs['type']]
-                if kwargs.get('default') is not True:
-                    kwargs.pop('default', None)
```

Comment:
I would've thought this (and the lines above it) would have been sufficient to clear out the defaults?